### DISCUSSION TITLE
Upgrading to scikit-learn pypi package

### DISCUSSION THREAD

**Discussion by aElizaF** [dynamicslab/pysindy#463] [Ideas]
Hello, 
When going through a fresh install, the sklearn package is depreciated and causes many errors when trying to download from sklearn-0.0.postX.tar.gz. Is there any plans to replace sklearn with it's upgrade, scikit-learn?
(X is placeholder for 12, 11, 10, 9, 7, 5, 4, 1)

**Comment by Jacob-Stevens-Haas** [comment]
It's been scikit-learn for a long time.  The current requirement is for `"scikit-learn>=1.1"`, not sklearn.  I don't know how you managed to install sklearn.  Maybe some more detail about what you tried to do?
